Source/JavaScriptCore/dfg/DFGSpeculativeJIT.cpp

```diff
@@ -255,6 +255,79 @@ MacroAssembler::Jump SpeculativeJIT::emitOSRExitFuzzCheck()
     return result;
 }
 
+void SpeculativeJIT::cachedGetById(
```

Comment:
Let's revert these changes. I think this is not necessary to this fix. We should make this patch simple.

Reply:
And I think FTL change is also required.